models/models.js

```diff
@@ -0,0 +1,37 @@
+const db = require("./../db/connection.js");
+const format = require("pg-format");
+exports.selectReviews = () => {
+  return db
+    .query("SELECT * FROM reviews ORDER BY created_at DESC")
+    .then((reviews) => {
+      return reviews.rows;
+    })
+    .then((reviews) => {
+      const dbQuery = `SELECT
+      review_id,
+      COUNT(*)
+      FROM
+      comments
+      GROUP BY
+      review_id;
+      `;
+
+      return Promise.all([db.query(dbQuery), reviews]);
+    })
```

Comment:
Rather than adding the comment count in this way, I would look to do this doing an SQL join instead. This should do all the work for you.